cinema/views.py

```diff
@@ -1,7 +1,9 @@
 from datetime import datetime
 
 from django.db.models import F, Count
-from rest_framework import viewsets
+from rest_framework import viewsets, mixins
+from rest_framework.authentication import TokenAuthentication
```

Comment:
Consider using `SessionAuthentication` along with `TokenAuthentication` if you want to support both session and token-based authentication. This is not a critical issue but could enhance flexibility.